game-samples/src/golang/profile-service/models/player_test.go

```diff
@@ -0,0 +1,67 @@
+package models
+
+import "testing"
+
+func TestValidEmails(t *testing.T) {
+	var tests = []string{"good@gmail", "good@gmail.com", "good@somedomain"}
+
+	for _, email := range tests {
+		res := validateEmail(email)
+
+		t.Logf("Testing '%s'", email)
+
+		if res == false {
+			t.Error(
+				"For", email,
+				"expected", false,
```

Comment:
Should `good@gmail` be considered valid? Also, should this not be `"expected", true`?

Reply:
Re: `"expected", true` yes it should!  Re: "Should good@gmail be considered valid", This leverages the golang `net/mail` to parse an address. There's some discussion on various [SO questions](https://stackoverflow.com/questions/66624011/how-to-validate-an-email-address-in-go) around localdomains being technically valid email addresses rather than implementing a custom solution. The intent with me going this route was to eventually introduce a 'email_validated' check, where the signup service triggers a validation process.